vizro-ai/mkdocs.yml

```diff
@@ -3,13 +3,16 @@ site_url: https://vizro-ai.readthedocs.io/en/latest/
 nav:
   - Vizro-AI: index.md
   - Tutorials:
-      - Install Vizro-AI: pages/user-guides/install.md
       - A first example: pages/tutorials/quickstart.md
   - How-to guides:
-      - Run methods: pages/user-guides/run-vizro-ai.md
-      - Create advanced charts: pages/user-guides/create-advanced-charts.md
-      - Use different languages: pages/user-guides/use-different-languages.md
-      - Use Vizro-AI charts in dashboards: pages/user-guides/add-generated-chart-usecase.md
+      - FUNDAMENTALS:
+          - Install Vizro-AI: pages/user-guides/install.md
```

Comment:
Have we checked that this does not break the tutorial flow?

Reply:
Yes, it's just a change in MKDocs because it was already in the guides folder and the self-contained installation page is separate from the tutorial, which refers to it in (it's basically step 1). So all good.